### DISCUSSION TITLE
captured data packets number are sometimes a better noise indicator

### DISCUSSION THREAD

**Discussion by e3k** [VREMSoftwareDevelopment/WiFiAnalyzer#372] [Ideas]
Initially i have used airodump-ng to analyse wifi situations. Then i learned WifiAnalyzer on Android and it all got simpler.
Currently i was investigating a wifi issue at my place with both airodump-ng and WifiAnalyser and noticed an interesting information aspect of airodump-ng. The APs causing trouble was also submitting masses of data which is clearly displayed in airodump-ng.

I guess WifiAnalyzer is only capturing BSSID broadcast and measuring the DB. A more precise strategy is to pinpoint the trouble making AP by measuring received intact data packets number. Higher data packet count means more noise.

This tactic is not yet implemented in WifiAnalyzer that is why i resort to airodump-ng.

**Comment by VREMSoftwareDevelopment** [comment]
This is a feature request, not a discussion
